src/App/index.jsx

```diff
@@ -81,7 +85,8 @@ export default class App extends Component {
                     path={path}
                     exact={exact}
                     render={({ staticContext, ...renderProps }) => (
-                      <Suspense fallback={<Loading />}>
+                      <Suspense
+                        fallback={<Spinner className={classes.spinner} />}>
```

Comment:
Are we supposed to provide the `loading` prop to cover for the old style `textAlign: 'center',`?

```js
fallback={<Spinner loading ... />}>
```

Reply:
I think it is not needed. When I test the spinner, it looks centered.
<img width="1443" alt="screen shot 2019-02-14 at 2 13 13 pm" src="https://user-images.githubusercontent.com/24544912/52766659-cfa2ec00-3062-11e9-8b9e-d72c3892b7f9.png">